cli/job.go

```diff
@@ -69,7 +71,28 @@ func runJobCmd() *cobra.Command {
 			logger := log.NewCtxLogger(config.LogLevel, []string{config.AuditLogTraceIDHeaderKey})
 			crypto := crypto.NewAES(config.EncryptionSecretKeyKey)
 			validator := validator.New()
-			notifier, err := notifiers.NewClient(&config.Notifier, logger)
+			var notifierMap map[string]interface{}
```

Comment:
This code returns error if `Notifiers` config is not present in configuration.
Therefore misses the backward compatibility.
Need to provide empty {} Json to `Notifiers` in configuration to make it work, But it finally doesn't send notification with old `Notifier` config